courses/pydata/info.yml

```
@@ -0,0 +1,21 @@
+title: Kurz datové analýzy
+description: Naučte se využít Python a nástroje pro něj napsané k datové analytice
+long_description: |
+
+    Materiály pro kurz datové analytiky v Pythonu. Materiály předpokládají znalost
+    Pythonu na úrovni absolventů začátečnického kurzu (PyLadies).
+
+canonical: true
+vars:
+  coach-present: false
+plan:
+- title: Instalace a práce s notebookem
+  slug: install
+  materials:
+  - lesson: pydata/install
+  - lesson: pydata/notebook
+
+- title: Práce s tabulkovými daty
```

```suggestion
- title: Základy práce s tabulkovými daty
```

To se bude dělat i v dalších hodinách, tak aby to nepůsobilo hned děsivě.